frontend/src/pages/Home.tsx

```diff
@@ -206,7 +213,10 @@ export default function Home() {
               ))}
             </div>
           </SecondaryCard>
-          <SecondaryCard title="New Projects" className="overflow-hidden">
+          <SecondaryCard
+            title={<AnchorTitle href="#new-Projects" title="New Projects" />}
```

Comment:
Sorry I missed this before! 

```suggestion
            title={<AnchorTitle href="#new-projects" title="New Projects" />}
```